### PR TITLE
Work with Support to refine SED documentation.

### PR DESCRIPTION
**Pull request by Mrt134**
cherry-pick PR #824 to 11.2-angular.

### CODE REVIEW

File: userguide/system.rst [hunk 1]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+important to use :command:`sedutil-cli` rather than camcontrol to access
+the full capabilities of the device. %brand% provides the
+:command:`sedhelper` wrapper script to ease SED administration from the
+command line.
 
-When SED devices are detected during system boot, the middleware checks
-for global and device-specific passwords. Devices with their own password
-are unlocked with their password and any remaining devices, without a
-device-specific password, are unlocked using the global password.
+By default, SED are not locked until the administrator takes ownership
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
By default, SEDs are not locked until the administrator takes ownership
```

File: userguide/system.rst [hunk 2]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+of them. This is done by explicitly configuring a global or per-device
```
Comment (wblock) [inline userguide/system.rst]: Nonspecific "this", maybe reword:
```suggestion
of them. Ownership is taken by explicitly configuring a global or per-device
```

File: userguide/system.rst [hunk 3]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+password in the %brand% |web-ui| and adding the password to the SEDs.
 
-To configure a global password, go to :menuselection:`System -->
-Advanced --> SED Password` and enter the
-password. Recording the password and storing it in a safe place is
-recommended.
+A password-protected SED protects the data stored on the device
+when the device is physically removed from the %brand% system. This
+allows secure disposal of the device without having to first wipe its
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
allows secure disposal of the device without having to first wipe the
```

File: userguide/system.rst [hunk 4]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+contents. If the device is instead removed to be repurposed on another
```
Comment (wblock) [inline userguide/system.rst]: The "instead removed" makes the second sentence confusing.  Well, that, and it's hard to see how "if the password is known" relates to the first sentence.  Please clarify.  Also, the second sentence could be restructured without the if/then pause: "SEDs that will be repurposed on another system can only be unlocked if the password is known."

File: userguide/system.rst [hunk 5]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
 
-.. code-block:: none
+.. _Deploying SEDs:
 
- sedutil-cli --scan
+Deploying SEDs
+^^^^^^^^^^^^^^
 
-In the results:
+To determine which devices support SED and their device names, go to the
```
Comment (wblock) [inline userguide/system.rst]: Flip sentence around to avoid if/then (to do this, do that).  Use :ref: for Shell to give them a link.
`` Run :command:`sedutil-cli --scan` in the :ref:`Shell` to detect and list devices. The second column
of the results identifies the drive type: ``

File: userguide/system.rst [hunk 6]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+   /dev/da0   No  HGST    HUS726020AL4210  A7J0
+   /dev/da1   No  HGST    HUS726020AL4210  A7J0
+   /dev/da10    E WDC     WUSTR1519ASS201  B925
+   /dev/da11    E WDC     WUSTR1519ASS201  B925
+
+
+%brand% supports setting a global password for all detected SEDs or
+setting individual passwords for each SED. Using a global password for
+all SEDs is strongly recommended to simplify deployment and avoid
+physically maintaining separate passwords for each SED.
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
maintaining separate passwords for each SED.
```

File: userguide/system.rst [hunk 7]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+
+
+**Setting a global password for SEDs**
```
Comment (wblock) [inline userguide/system.rst]: This should be a section, not just bold.

File: userguide/system.rst [hunk 8]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+
+Go to
+:menuselection:`System --> Advanced --> SED Password`
+and enter the password. **Record this password and store it in a safe
+place!**
+
+Now the SEDs must be configured with this password. Go to the
+:menuselection:`Shell`
```
Comment (wblock) [inline userguide/system.rst]: Use :ref: for Shell to link them to the Shell section.

File: userguide/system.rst [hunk 9]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+and enter :samp:`sedhelper setup {password}`, where *password* is the
+global password entered in
+:menuselection:`System --> Advanced --> SED Password`.
 
-Remember to take ownership of the devices:
+:command:`sedhelper` ensures that all detected SEDs are properly
+configured to use the provided password. Example:
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
configured to use the provided password:
```

File: userguide/system.rst [hunk 10]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+   da9			[OK]
+   da10			[OK]
+   da11			[OK]
+
+
+Rerun :samp:`sedhelper setup {password}` every time a new SED is placed
+in the system to apply the global password to the new SED.
+
+
+**Creating separate passwords for each SED**
```
Comment (wblock) [inline userguide/system.rst]: Section header.

File: userguide/system.rst [hunk 11]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+
+Go to
+:menuselection:`Storage --> Disks`.
+Click |ui-options| for the confirmed SED, then :guilabel:`Edit`.
+Enter and confirm the password in the :guilabel:`SED Password` and
+:guilabel:`Confirm SED Password` fields. Disks that have a configured
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
:guilabel:`Confirm SED Password` fields. Disks with a configured
```

File: userguide/system.rst [hunk 12]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+SED password show bullets in their row of the :guilabel:`SED Password`
```
Comment (wblock) [inline userguide/system.rst]: "show bullets" is... odd. 
The menu entry should be first, because they have to go there before seeing the column.
Then:
"...are indicated by a mark in the :guilabel:`SED Password`"

File: userguide/system.rst [hunk 13]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+column of
+:menuselection:`Storage --> Disks`.
+Conversely, the rows in that column will be empty for disks that do not
```
Comment (wblock) [inline userguide/system.rst]: Please don't do this.  It's the same as "alternatively", a negative that forces the reader to stop and think oppositely.  It is not needed here:
"Disks that do not support SED are not marked in this column."

"Support SED" is still an odd way to word it ("support self-encrypting devices" is misleading).  It can probably be said differently, like "support Opal encryption" or something.  We need to research this.

File: userguide/system.rst [hunk 14]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+support SED or are unlocked using the global password.
+
+Now configure the SED to use the password. Go to the
```
Comment (wblock) [inline userguide/system.rst]: Rewrite to avoid Now, like "The next step is to..."

File: userguide/system.rst [hunk 15]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+:menuselection:`Shell`
```
Comment (wblock) [inline userguide/system.rst]: :ref:

File: userguide/system.rst [hunk 16]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+and enter :samp:`sedhelper setup --disk {da1} {password}`, where *da1*
+is the SED to configure and *password* is the created password from
+:menuselection:`Storage --> Disks --> Edit Disks --> SED Password`.
+
+**Repeat this process for each detected SED and any SED deployed to the
```
Comment (wblock) [inline userguide/system.rst]: This process must be repeated for each SED and any SEDs add to the system in the future.

I don't think bold is needed here.

File: userguide/system.rst [hunk 17]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+%brand% system in the future.**
+
+.. danger:: It is important to remember SED passwords! Without them,
```
Comment (wblock) [inline userguide/system.rst]: Use an imperative: SED passwords *must* be remembered. If the SED password is lost,

File: userguide/system.rst [hunk 18]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+   SEDs cannot be unlocked and their data is unavailable. While it is
+   possible to specify the PSID number on the label of the device with
+   :command:`sedutil-cli`, doing so **erases the contents** of the
+   device rather than unlock it. Always record SED passwords whenever
+   they are configured or modified and store them in a safe place!
```
Comment (wblock) [inline userguide/system.rst]: s/safe/secure/

File: userguide/system.rst [hunk 19]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
 * TCG Enterprise standard for newer enterprise-grade SAS devices
 
 The %brand% middleware implements the security capabilities of
```
Comment (wblock) [inline userguide/system.rst]: We should avoid mentioning middleware as a separate component.  As William would say, it's an implementation detail.

File: userguide/system.rst [hunk 20]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
-This command ensures that all detected SED disks are properly setup using
-the specified password.
+Check SED Functionality
+^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note:: Rerun :command:`sedhelper setup password` every time a new SED
-   disk is placed in the system.
+When SED devices are detected during system boot, the middleware
+checks for configured global and device-specific passwords. Devices with
+individual passwords are unlocked with their password and any remaining
```
Comment (wblock) [inline userguide/system.rst]: Split long sentence after "password".  "Any remaining" is not needed.

File: userguide/system.rst [hunk 21]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+devices that have no device-specific password are unlocked using the
```
Comment (wblock) [inline userguide/system.rst]: Devices without a device-specific password...

File: userguide/system.rst [hunk 22]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+global password. No additional manual intervention is required, but
```
Comment (wblock) [inline userguide/system.rst]: The second and following sentences should be split into a separate paragraph about using a mixed pool.

File: userguide/system.rst [hunk 23]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+:command:`sedhelper unlock` is available to manually unlock all SEDs.
+Unlocking SEDs allows a pool to contain a mix of SED and non-SED
```
Comment (wblock) [inline userguide/system.rst]: Is this something we want to implicitly recommend?

File: userguide/system.rst [hunk 24]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+devices.
 
-This command is used to unlock all available SED disks:
+To verify SED locking is working correctly, go to the
+:menuselection:`Shell`. Enter
```
Comment (wblock) [inline userguide/system.rst]: :ref:

File: userguide/system.rst [hunk 25]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+:samp:`sedutil-cli --listLockingRange 0 {password} dev/{da1}`, where
+*da1* is the SED and *password* is the global or individual password for
+that SED. The command returns :literal:`ReadLockEnabled: 1`,
+:literal:`WriteLockEnabled: 1`, and :literal:`LockOnReset: 1` when the
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
:literal:`WriteLockEnabled: 1`, and :literal:`LockOnReset: 1` for drives with
```

File: userguide/system.rst [hunk 26]
```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
...
+SED has locking enabled. Example:
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
locking enabled:
```

File: userguide/system.rst [hunk 27]
```diff
@@ -738,88 +739,158 @@ Three types of SED devices are supported:
...
+password. Devices without a device-specific password are unlocked using
+the global password.
 
-This command is used to unlock all available SED disks:
+To verify SED locking is working correctly, go to the :ref:`Shell`.
+Enter :samp:`sedutil-cli --listLockingRange 0 {password} dev/{da1}`,
+where *da1* is the SED and *password* is the global or individual
+password for that SED. The command returns :literal:`ReadLockEnabled: 1`,
+:literal:`WriteLockEnabled: 1`, and :literal:`LockOnReset: 1` for drives
+locking enabled:
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
with locking enabled:
```
